src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,76 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private boolean isInitialRead = true;
```

Comment:
```suggestion
```
What is it for?

Reply:
It's needed to provide reading file once. Because the las one of tests has multiple invoke of method getstatistic and tnen i had a duplication of final result, so this test was failed.

Reply:
It's needed to provide reading file once. Because the las one of tests has multiple invoke of method getstatistic and tnen i had a duplication of final result, so this test was failed.

Reply:
This variable is useless. The result should be rewritten. That's it. To do non-reusable class - very bad practice. Fix it